### ISSUE TITLE
slice viewer in interactive mode

### ISSUE THREAD

**Issue by pelednoam** [nipy/nibabel#432]
Hey,
Is there a way to open to slice viewer in an interactive mode?
I want to open it from my program, and let the user the option to click on a button from my program which will change the position.

Thanks,
Noam

**Comment by matthew-brett** [comment]
As far as I know this will work in the same way as any other matplotlib
figure. I think you want to enable "interactive mode" :
http://matplotlib.org/faq/usage_faq.html#interactive-example

Does that work?

**Comment by pelednoam** [comment]
Yes, this is a known issue with matplotlib. In my framework I need to open
the slice viewer in its own thread and open a process using the shell, and
it seems that the interactive mode doesn't work when you do something like
that. I've tried all the tricks I found in stackoverflow...
The only solution I've found was to close the viewer and reopen it in the
new location, but this is quite a bad solution.
Thanks anyway, this viewer was exactly what I was looking (except of the
matplotlib problems...)

Best,
Noam
